courses/models.py

```
@@ -1,25 +1,35 @@
+
 from core.models import BaseModel
 from accounts.models import User
 
 
-class Classroom(BaseModel):
+class Course(BaseModel):
+    LEVEL_CHOICES = [
+        ('beginner', 'Beginner'),
+        ('intermediate', 'Intermediate'),
+        ('advanced', 'Advanced'),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)